pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -398,6 +391,8 @@ func (configgen *ConfigGeneratorImpl) createGatewayHTTPFilterChainOpts(
 				ServerName:          EnvoyServerName,
 				HttpProtocolOptions: httpProtoOpts,
 			},
+			addGRPCWebFilter: serverProto == protocol.GRPCWeb,
```

Comment:
For Http, we seems to use a single http connection manager - So I am not setting the stat prefix there.